backend/src/routes.js

```diff
@@ -24,14 +26,24 @@ router.post("/products", async (req, res, next) => {
   }
 })
 
-router.post("/orders", async (req, res) => {
-  const { products } = req.body
-  for (let i = 0; i < products.length; i++) {
-    const product = await Product.findById(new mongoose.Types.ObjectId(products[i])).lean()
-    products[i] = product
+mercadopago.configure({
+  access_token: process.env.MERCADOPAGO_ACCESS_TOKEN,
+})
+
+router.post("/orders", async (req, res, next) => {
+  const products = req.body
+  try {
+    for (let i = 0; i < products.length; i++) {
+      const product = await Product.findById(new mongoose.Types.ObjectId(products[i])).lean()
+      products[i] = product
+    }
+    const items = products.map((product) => ({ title: product.name, unit_price: product.price, quantity: 1 }))
```

Comment:
veo que acá usas map pero arriba usas el `for` estándar creo que es mejor tener consistencia de código y usar una de las dos.

Reply:
también considero que deberías pasarle el quantity que no sea siempre uno